src/sequencer.rs

```diff
@@ -17,11 +17,10 @@ pub struct Client {
     sequencer_url: Url,
 }
 
-/// Helper function which simplifies the handling of optional block IDs in queries.
-fn block_id_str(block: Option<U256>) -> String {
-    block
-        .map(|b| b.to_string())
-        .unwrap_or_else(|| "null".to_string())
+/// Helper function which simplifies the handling of optional block hashes in queries.
+fn block_hash_str(hash: Option<H256>) -> (String, String) {
```

Comment:
Could consider `(&'static str, Cow<'static, str>)` for the return type. These seem to be used as &str so should be good.